src/kitsController.ts

```diff
@@ -580,4 +552,49 @@ export class KitsController {
 
         return duplicateRemoved;
     }
+
+    static isBetterMatch(newKit: Kit, existingKit?: Kit): boolean {
+        if (KitsController.isBetterClangCLDefinition(newKit, existingKit)) {
+            return true;
+        }
+        return KitsController.isBetterCompilerMatch(newKit.compilers, existingKit?.compilers);
+    }
+
+    static isBetterClangCLDefinition(newKit: Kit, existingKit?: Kit): boolean {
+        if (newKit.name.indexOf('MSVC CLI') >= 0) {
```

Comment:
IS this part to fix the kits we created previously with "amd64" in their fields?

Reply:
yes. If we don't do this, the old, incorrect kits will stay around forever.  Even after rescanning.

Reply:
We might even want to be more aggressive in the future, detect the bad kits and automatically do the rescan.